add object_details to information_schema.columns (#11744)
crate/crate: docs/appendices/release-notes/unreleased.rst

  session settings, unless the client sets the session settings on each node
  individually.

> - Added `object_details` to the `information_schema.columns` table including the
>   top level column for objects, path information and leaf name

Reviewer: ```suggestion
- Added the `column_details` column to the `information_schema.columns` table 
  including the top level column name and path information of object elements.
```

Author: 🤭 ooops
